drv/i2c-devices/src/mwocp68.rs

```diff
@@ -62,6 +115,43 @@ impl From<pmbus::Error> for Error {
     }
 }
 
+// const MWOCP68_MFR_ID: &str = "Murata-PS";
+// const MWOCP68_MFR_MODEL: &str = "MWOCP68-3600-D-RM";
+
+const MWOCP68_REVISION_LEN: usize = 14;
```

Comment:
Nit: these could live in `firmware_revision` instead of the global namespace, since that's where they're used.